server/routes/topic_page.py

```diff
@@ -36,6 +37,8 @@ def topic_page(topic_id=None, place_dcid=None):
         return flask.render_template('topic_page_landing.html')
 
     all_configs = current_app.config['TOPIC_PAGE_CONFIG']
+    if os.environ.get('FLASK_ENV') == 'local':
+        all_configs = libutil.get_topic_page_config()
```

Comment:
curious why you're getting all_configs here. Does local flask environment not have the configs saved in the app config?

Reply:
i was making many updates to the configs and wanted a hot reload (instead of fully restarting the server). so locally, you can separately compile the protos and the latest config will be used